doc/.vale.ini

```diff
@@ -0,0 +1,28 @@
+# Core settings
+# =============
+
+# Location of our `styles`
+StylesPath = "styles"
+
+# The options are `suggestion`, `warning`, or `error` (defaults to “warning”).
```

Comment:
```suggestion
# The options are ``"suggestion"``, ``"warning"``, and ``"error"``. The default is ``“warning”``.
```

Reply:
This might be open to debate as the subsequent lines are just using a single back tick. Do they differ in what they mean? We should be consistent.

Reply:
This is the configuration file for Vale. Thus, it is not expected to be rendered in our docs. I just copy-pasted what we have in other projects.

I guess we should discard these changes so the whole configuration file is uniform and similar to other config files across projects.